package.json

```diff
@@ -2,14 +2,14 @@
   "name": "react-native-charts-wrapper",
   "repository": {
     "type": "git",
-    "url": "https://github.com/wuxudong/react-native-charts-wrapper.git"
+    "url": "https://github.com/ramki1979/react-native-charts-wrapper.git"
```

Comment:
Realy?

Reply:
Sorry, this is just for my Bitrise CI workflow other wise the build is getting build from your branch and crashing.